### PR TITLE
1311/adding utility options

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses #1311. 

This PR adds the option (not required) to indicate which utilities are included in the cost of rent. This is seen on the partners form fields, the listing detailed view on the partners side, and the detailed listing view on the public side. The public side representation in the AdditionalFees component required a prop refactoring so that a user can pass in an array of ReactNodes or strings to be displayed in a 2 column-format. Also, I expanded a few info-card__column css classes so that all sections of AdditionalFees.tsx resize in a consistent way and so that the classes could be potentially used for dynamic column representations in components other than AdditionalFees.tsx.

On the backend, this PR required a migration to add the new 'utilities' table to the model along with multiple updates to various backend type files. Additionally, translations and tests were added to complete and verify these changes.

- [x] This change addresses the issue in full

## How Can This Be Tested/Reviewed?
Testing on Partners:
Part 1: This can be tested by selecting a listing, clicking edit, and scrolling down to ensure that the new utilities form fields are available and clickable. After selecting a variety of utilities, click 'save & exit', and then ensure that they are on the detailed view. 

Part 2: Go back to edit the same listing, unselect all options for utilities, and then save the listing again. The 'Utilities Included' title and bullets should be no longer visible.

Testing on Public:
Select a listing that you recently marked as having included utilities and observe the bulleted list in the 'Additional Fees' Component. If other 'additional fee' messages are available for the listing, the section will be in a 2-column format (with the exception of the last row which will fill if there is an odd number of footer info pieces).

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: ui-components/src/page_components/listing/AdditionalFees.stories.tsx
Comment (emilyjablonski) [inline ui-components/src/page_components/listing/AdditionalFees.stories.tsx:15]: Would like to see some stories added that showcase the new ability for the footer to have multiple content sections, probably as its own story.

File: ui-components/src/blocks/InfoCard.scss
Comment (emilyjablonski) [inline ui-components/src/blocks/InfoCard.scss:49]: Just curious what kinds of layout impacts resulted in `440`px?

File: sites/partners/src/listings/PaperListingDetails/sections/DetailAdditionalFees.tsx
Comment (emilyjablonski) [inline sites/partners/src/listings/PaperListingDetails/sections/DetailAdditionalFees.tsx:20]: We can use the `t.none` string here (looks existing in the features piece as well)

File: ui-components/src/page_components/listing/AdditionalFees.tsx
Comment (emilyjablonski) [inline ui-components/src/page_components/listing/AdditionalFees.tsx:5]: So this is the piece where it also needs to happen in core. We're trying to remove all changes in ui-components that are not in core so that we can centralize on one version. I think it's completely fine to make these ui-component changes in Detroit first, and have a separate PR in core after this is merged to bring these changes back in.

**Comment by emilyjablonski** [comment]
This looks excellent! Since we paired a little bit on the approach, would wait for a review from one other person.

File: ui-components/src/page_components/listing/AdditionalFees.stories.tsx
Comment (ColinBuyck) [reply ui-components/src/page_components/listing/AdditionalFees.stories.tsx:64]: Added!

**Review by YazeedLoonat — APPROVED** [review]
LGTM

**Pull request merged**
